.gitignore

```diff
@@ -1,4 +1,5 @@
 build
 dist
+src
```

Comment:
Because of `dev_requirements.txt`. When I do `pip install -r dev_requirements.txt`, Python will create a `src/` subfolder in the repo and then clone all the requirements and install them in developer mode from there.